tests/random_attack.py

```diff
@@ -224,6 +224,15 @@ def replace_nas_in_column(self, frame):
                                 % (icol, icol, replacement_value))
         frame.replace_nas_in_column(icol, replacement_value)
 
+    def sort_column(self, frame):
+        if frame.nrows == 0 or frame.ncols == 0:
```

Comment:
Sorting a 0-row frame is a valid operation, so why prohibit this case? 
Especially since it has a higher chance of uncovering an error...

Reply:
This case won't work with the existing python part. Need to adjust it.

Reply:
well you can do an early return in the python part

Reply:
Yeah, adding it now.

Reply:
Actually, even sorting on a zero-column frame will work. If one does `DT.sort()`. Not sure if this could be a useful use case though.